test/unit/Command/DownloadCommandTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Php\PieUnitTest\Command;
+
+use Php\Pie\Command\DownloadCommand;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[CoversClass(DownloadCommand::class)]
+class DownloadCommandTest extends TestCase
+{
+    public function testDownloadCommand(): void
+    {
+        $input  = $this->createMock(InputInterface::class);
+        $output = $this->createMock(OutputInterface::class);
```

Comment:
Just FYI as you may not be aware of it, there's a tester class for commands integration testing, see e.g.

https://github.com/composer/composer/blob/main/tests/Composer/Test/TestCase.php#L201-L211 (the ApplicationTester class is part of symfony/console tho)

Reply:
Thanks - I didn't know about this. I've switched to using `CommandTester` in a follow-up branch to this, so will be addressed then :+1: